### ISSUE TITLE
"No messages on server" flashes when first logging in

### ISSUE THREAD

**Issue by timabbott** [zulip/zulip-mobile#1230]
I just logged out and in again on chat.zulip.org after reinstalling the app, and there's a brief flicker of "No messages on server" before the app finishes fetching from the server.  

We should treat the state of "doing initial fetch" different from "nothing here" in the UI.

**Comment by borisyankov** [comment]
Indeed. Will fix next, should be a quickie.

**Comment by borisyankov** [comment]
@timabbott are you sure you have seen this on a recent build? We used to have this issue few versions ago and was fixed (or at least improved). I can not reproduce now.

**Comment by timabbott** [comment]
Yep saw it just after reinstalling the same day I reported it.

-Tim Abbott (mobile)

On Sep 30, 2017 13:41, "Boris Yankov" <notifications@github.com> wrote:

@timabbott <https://github.com/timabbott> are you sure you have seen this
on a recent build? We used to have this issue few versions ago and was
fixed (or at least improved). I can not reproduce now.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/zulip/zulip-mobile/issues/1230#issuecomment-333334640>,
or mute the thread
